### PR TITLE
Covid-19 Update 2

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: docs/conf/portland/2020/news/covid-19-update-02.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Hey Folks,
+
+Since our :doc:`first announcement </conf/portland/2020/news/covid-19-update-01>` last week, we've been following the COVID-19 updates from the Oregon Health Authority and the CDC. We've also seen `King County`_ (Seattle) and `Santa Clara County`_ (SF Bay Area) explicitly request organizations cancel or postpone large gatherings. This has made us come to the conclusion that our event cannot go on as currently scheduled.
+
+**Our current plan of action is to postpone the conference, most likely to a date in August.**
+
+We'll be talking to our venue, staff, speakers, and sponsors to find a date that works. We'd like to keep uncertainty to a minimum, so we hope to announce a firm date by **March 20** at the latest.
+
+Other options are a virtual conference, or canceling the Portland event entirely in 2020, and we don't think that either of these are ideal for Write the Docs.
```
Comment (Bradamant3) [inline docs/conf/portland/2020/news/covid-19-update-02.rst]: s/of these are/is (don't need "of these" at all)

File: docs/conf/portland/2020/news/covid-19-update-02.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+If you have any questions about this, please reach out to `{{email}} <mailto:{{email}}>`_.
```
Comment (Bradamant3) [inline docs/conf/portland/2020/news/covid-19-update-02.rst]: rm "about this"

**Comment by Bradamant3** [comment]
LGTM but not approving bc other eyes. would be good

File: docs/conf/portland/2020/news/covid-19-update-02.rst [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
+.. post:: March 11, 2020
    :tags: portland-2020, covid-19
 
 COVID-19 Update 02
 ==================
 
 Hey Folks,
 
-Since our :doc:`first announcement </conf/portland/2020/news/covid-19-update-01>` last week, we've been following the COVID-19 updates from the Oregan Health Authority and the CDC, as well as seeing a number of other organizations cancelling or postponing events due to be held in the next few months, and we've come to the reluctant conclusion that we also need to look into alternatives.
+Since our :doc:`first announcement </conf/portland/2020/news/covid-19-update-01>` last week, we've been following the COVID-19 updates from the Oregon Health Authority and the CDC. We've also seen `King County`_ (Seattle) and `Santa Clara County`_ (SF Bay Area) explicitly request organizations cancel or postpone large gatherings. This has made us come to the conclusion that our event cannot go on as currently scheduled.
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/covid-19-update-02.rst]: Trying to find a rewrite for "This has made us come to the conclusion "

File: docs/conf/portland/2020/news/covid-19-update-02.rst [hunk 4]
```diff
@@ -14,7 +14,7 @@ Since our :doc:`first announcement </conf/portland/2020/news/covid-19-update-01>
 
 We'll be talking to our venue, staff, speakers, and sponsors to find a date that works. We'd like to keep uncertainty to a minimum, so we hope to announce a firm date by **March 20** at the latest.
 
-Other options are a virtual conference, or canceling the Portland event entirely in 2020, and we don't think that either of these are ideal for Write the Docs.
+Our current plan would be to automatically transfer all existing tickets and sponsorships to the new conference date. We will also refund anyone 100% who can't make the new date work with their schedule.
 
 If you have any questions about this, please reach out to `{{email}} <mailto:{{email}}>`_.
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/covid-19-update-02.rst]:
```suggestion
If you have any questions, please reach out to `{{email}} <mailto:{{email}}>`_.
```

File: docs/conf/portland/2020/news/covid-19-update-02.rst [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Hey Folks,
+
+Since our :doc:`first announcement </conf/portland/2020/news/covid-19-update-01>` last week, we've been following the COVID-19 updates from the Oregon Health Authority and the CDC. We've also seen `King County`_ (Seattle) and `Santa Clara County`_ (SF Bay Area) explicitly request organizations cancel or postpone large gatherings. This has made us come to the conclusion that our event cannot go on as currently scheduled.
+
+**Our current plan of action is to postpone the conference, most likely to a date in August.**
+
+We'll be talking to our venue, staff, speakers, and sponsors to find a date that works. We'd like to keep uncertainty to a minimum, so we hope to announce a firm date by **March 20** at the latest.
+
+Our current plan would be to automatically transfer all existing tickets and sponsorships to the new conference date. We will also refund anyone 100% who can't make the new date work with their schedule.
```
Comment (plaindocs) [inline docs/conf/portland/2020/news/covid-19-update-02.rst]:
```suggestion
If we do reschedule, we'll transfer all existing tickets and sponsorships to the new conference date. We will refund tickets without charge for anyone who can't make the new date.
```

File: docs/conf/portland/2020/news/covid-19-update-02.rst (hunk 3 above)
Comment (plaindocs) [reply docs/conf/portland/2020/news/covid-19-update-02.rst]: "We have concluded"?

**Review by plaindocs — COMMENTED** [review]
LGTM

**Comment by mopdx** [comment]
Looks good.

**Pull request merged**